de.wolfvollprecht.UberWriter.Plugin.TexLive.json

```diff
@@ -0,0 +1,91 @@
+{
+    "id": "de.wolfvollprecht.UberWriter.Plugin.TexLive",
+    "runtime": "de.wolfvollprecht.UberWriter",
+    "branch": "stable",
+    "sdk": "org.gnome.Sdk//3.26",
+    "build-extension": true,
+    "separate-locales": false,
+    "appstream-compose": false,
+    "finish-args": [
+        ],
+    "build-options" : {
+        "prefix": "/app/extensions/TexLive",
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+          "PATH": "/app/extensions/TexLive/bin:/app/extensions/TexLive/2018/bin/x86_64-linux:/app/bin:/usr/bin"
+        }
+    },
+    "cleanup": ["/bin/wget"],
+    "modules": [
+        {
+            "name": "wget",
+            "builddir": true,
+            "config-opts": ["--prefix=/app/extensions/TexLive"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://ftp.gnu.org/gnu/wget/wget-1.19.4.tar.gz",
+                    "sha256": "93fb96b0f48a20ff5be0d9d9d3c4a986b469cb853131f9d5fe4cc9cecbc8b5b5"
+                }
+            ]
+        },
+        {
+            "name": "p7zip",
+            "no-autogen": true,
+            "config-opts": ["--prefix=/app/extensions/TexLive"],
+            "make-args": ["7z"],
+            "cleanup": ["*"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://sourceforge.net/projects/p7zip/files/p7zip/16.02/p7zip_16.02_src_all.tar.bz2",
+                    "sha256": "5eb20ac0e2944f6cb9c2d51dd6c4518941c185347d4089ea89087ffdd6e2341f"
+                },
+                {
+                    "type": "shell",
+                    "commands": ["sed -i 's|/usr/local|/app/extensions/TexLive|g' makefile.common"]
+                }
+            ]
+        },
+        {
+			"name": "texlive",
+			"buildsystem": "simple",
+			"sources": [
+				{
+					"type":"file",
+					"url": "http://mirrors.ctan.org/systems/texlive/Images/texlive.iso",
+					"sha512": "7b7f0dd0eab3bfffe52c5cd1139c7f75d029b9ff4c4ce0e57e06834705522f4ec0c02cd99a80b053c6619abda51c20a60f8e91e91781bdc2b9b60fc2e5708adb"
+				},
+				{
+			                "type": "file",
+	                                "path": "texlive.profile"
+             			},
+				{
+                    "type": "shell",
+                    "commands": ["7z x texlive.iso"]
+				}
+			],
+			"build-commands": [
+				"chmod +x install-tl",
+				"echo \"In\" | TEXLIVE_INSTALL_PREFIX=/app/extensions/TexLive ./install-tl --profile texlive.profile",
+				"yes | rm -r /app/extensions/TexLive/texmf-dist/doc"
```

Comment:
Remove `yes`?

Reply:
probably `rm -rf /app/...`  is better, I agree.

Reply:
Done: 8a63b4a2e08fea2145c8c3f4624e02803a8f76a8